lib/EntryLang.php

```diff
@@ -77,7 +80,10 @@ public function setCode(string $value): self
      */
     public function getName(): string
     {
-        return $this->getValue('name');
+        if ($this->hasValue('name')) {
+            return $this->getValue('name');
+        }
+        return '';
```

Comment:
Wäre man an diesen Stellen nicht kürzer hiermit?

```suggestion
      return $this->getValue('name') ?? '';
```

Reply:
Na ja, tw. Geschmackssache. Meine Gründe sind:
1) ist es besser lesbar (imho und der anderer)
2) meckert RexStan gerne mal ternäre Operatoren an, wehalb ich sie ebenfalls zunehmend vermeide
3) Hauptgrund: `$this->getValue('name') ?? ''` ist was anderes als `if($this->hasValue(..)){ ...}`, denn `hasValue`vermeidet die "Undefined Array Key"-Warnung im Fall dass das Elment doch noch nicht im data-Array steht.

Reply:
Wann könnte es denn zu einem "undefined array key" kommen? Wenn das entsprechende Feld bei den ausgelieferten Tablesets nicht (mehr) verfügbar ist, dann ist das ja ein grundsätzlicheres Problem.